execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -85,13 +88,16 @@ pub struct WasmTestBuilder<S> {
     /// [`EngineState`] is wrapped in [`Rc`] to work around a missing [`Clone`] implementation
     engine_state: Rc<EngineState<S>>,
     /// [`ExecutionResult`] is wrapped in [`Rc`] to work around a missing [`Clone`] implementation
-    exec_results: Vec<Vec<Rc<ExecutionResult>>>,
+    exec_results: Vec<Vec<(DeployHash, Rc<ExecutionResult>)>>,
+    /// Upgrade results.
     upgrade_results: Vec<Result<UpgradeSuccess, engine_state::Error>>,
+    /// .
     genesis_hash: Option<Digest>,
+    ///
```

Comment:
Can you fill in these comments with actual content?

Reply:
Done